test/node-unit/cli/config.spec.js

```diff
@@ -17,6 +16,17 @@ describe('cli/config', function() {
   });
 
   describe('loadConfig()', function() {
+    let parsers;
+    let loadConfig;
+
+    beforeEach(function() {
+      const config = rewiremock.proxy(
```

Comment:
This is confusing, this `config` shadowing the one of line 8?

Reply:
addressed this